requirements/base.in

```diff
@@ -1,7 +1,6 @@
 black~=23.0
 commitment<4.0.0
 dj-database-url
-django-apiblueprint-view==2.4.0
```

Comment:
Always in favour of deleting code! However is it desirable that the API is undocumented even if it's internal? 
I assume the rational is that we don't keep the docs up to date (guilty). However it still gives a sense of the endpoints and how they change depending on what elections are going on. Presumably that is useful for future onboarding etc.

Reply:
Maybe this is better done via code comments or readme files? Especially as it's not possible for anyone on a Mac to contribute to these docs as is?

Reply:
Just FYI

In version 2.5.0 and above, I now distribute a wheel with libdrafter binary compiled for `macosx_x86_64`. That might be runnable on `macosx_arm64` (m1/m2) macs via rosetta, but also I have no way to test that.

The conclusion I came to when I was working on that was that there is no way I could cross-compile an `arm64`-compatible version of libdrafter from a `macosx_x86_64` GH actions runner using the current C toolchain. It may be possible (but untested) to compile a `macosx_arm64` compatible binary on a `macosx_arm64` GH actions runner, but at the moment you can only use `macosx_x86_64` runners for free. Running an `macosx_arm64` is currently a premium feature, and it isn't something I particularly have the appetite to pay for.

https://github.com/chris48s/django-apiblueprint-view/pull/182